docs/apis/deleteEntity.md

```diff
@@ -0,0 +1,60 @@
+### `deleteEntity( resource: object ): Promise`
+
+Dispatch this action to issue a `DELETE` request to your API.
+
+### Endpoint
+
+Endpoint calculation for `deleteEntity` is the same as for `[updateEntity](./updateEntity.md)`. It is resolved from `type` and `id` of the resource object passed to this action creator.
```

Comment:
the Link `[updateEntity](./updateEntity.md)` is not rendered due to the backticks